### DISCUSSION TITLE
Using 2 frames of LIDAR to get acceleration information

### DISCUSSION THREAD

**Discussion by All-Ki** [trackmania-rl/tmrl#12] [Ideas]
Wouldnt using the previous and current 'frames' of LIDAR make the network able to extract speed information ?
That wouldnt be cheating per se since us humans also use our memory !

**Comment by yannbouteiller** [comment]
That is already what the environment does by default, it uses the 4 most recent screenshots so that the policy can infer its first order moments (speed/acceleration...) :)

**Reply by All-Ki** [reply]
Oh okay , i shouldve read the code better ^^
